README.md

```diff
@@ -5,11 +5,25 @@
 [![Test Coverage](https://codeclimate.com/github/18F/federalist/badges/coverage.svg)](https://codeclimate.com/github/18F/federalist/coverage)
 [![Dependency Status](https://gemnasium.com/badges/github.com/18F/federalist.svg)](https://gemnasium.com/github.com/18F/federalist)
 
-***Under active development. Everything is subject to change. Learn more at the [documentation site](https://federalist-docs.18f.gov/). Interested in talking to us? [Join our public chat room](https://chat.18f.gov/).***
+***Federalist is updated regularly. [Join our public chat room](https://chat.18f.gov/) to talk to us and stay informed. You can also check out our [documentation](https://federalist-docs.18f.gov/) to learn more.***
 
-Federalist is a unified interface for publishing static government websites. It automates common tasks for integrating GitHub and Amazon Web Services to provide a simple way for developers to launch new static websites or more easily manage existing static websites. This repo is home to "federalist-core" - a Node.js app that allows government users to add and configure their Federalist sites.
+## About Federalist
 
-## How to set up a local Federalist development server
+[Federalist](https://federalist.18f.gov) helps federal government entities publish compliant static websites quickly and seamlessly. Federalist is built on top of [cloud.gov](cloud.gov), GitHub, and Amazon Web Services.
```

Comment:
should we be a little more precise that Federalist integrates with Github?

Reply:
I think this is ok, but what about something like

"Federalist is built on top of cloud.gov and Amazon Web Services, and integrates with GitHub."

Reply:
switched it around a bit to clarify that cloud.gov uses AWS and we get AWS via cloud.gov, otherwise is 👍

Reply:
nice